src/status_im2/contexts/quo_preview/loaders/skeleton_list.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.contexts.quo-preview.loaders.skeleton-list
```

Comment:
why another preview screen?

Reply:
It was because re renamed to skeleton-list. The other one will be removed (or deprecated)

Reply:
Please remove unused preview screen

Reply:
we still have two skeleton preview screens.